platforms/ide/problems-rendering/src/main/java/org/gradle/problems/internal/rendering/package-info.java

```diff
@@ -13,25 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
-package org.gradle.composite;
-
-import org.gradle.internal.problems.failure.Failure;
-import org.gradle.internal.service.scopes.Scope;
-import org.gradle.internal.service.scopes.ServiceScope;
-import org.jspecify.annotations.NullMarked;
-
-import java.util.Collection;
-
-@ServiceScope(Scope.BuildTree.class)
 @NullMarked
-public interface ResilientIssuesRecorder {
+package org.gradle.problems.internal.rendering;
```

Comment:
🤔(Commenting here as this is a package concern) WDYT if you would move the actual rendering bits into a sub-package? E.g. `renderers`. I find navigating this package a bit of a hassle, it's hard to see what different renders there are